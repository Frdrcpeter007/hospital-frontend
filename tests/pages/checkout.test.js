import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Page from '../../pages/checkout';
require('jest-fetch-mock').enableMocks();

fetch.mockResponse(JSON.stringify({ clientSecret: 'test' }));

describe('Checkout Page', () => {
  it('renders', () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
