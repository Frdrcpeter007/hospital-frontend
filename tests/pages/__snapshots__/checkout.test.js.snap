// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkout Page renders 1`] = `
<div>
  <div
    class="App"
  />
</div>
`;
